lsp-ui-imenu.el

```diff
@@ -112,62 +149,120 @@
         (when (overlayp lsp-ui-imenu-ov)
           (delete-overlay lsp-ui-imenu-ov))))))
 
+(defun lsp-ui-imenu--move-to-line-beginning ()
```

Comment:
Is there a better name for this function? This actually moves to an indented position.

Reply:
Renamed

Reply:
Renamed